packages/syft/src/syft/server/server.py

```diff
@@ -410,9 +412,12 @@ def __init__(
                 filename=f"{self.id}_json.db",
                 path=self.get_temp_dir("db"),
             )
-            # db_config = PostgresDBConfig(reset=False)
+
+        if reset:
+            db_config.reset = True
 
         self.db_config = db_config
+        self.db: PostgresDBManager | SQLiteDBManager | None = None
```

Comment:
should be `self.db: DBManager` (and not nullable)

perhaps:
```
self.db = self.init_db(db_config)
```

Reply:
Resolved in [000092d](https://github.com/OpenMined/PySyft/pull/9284/commits/000092d78a33e14ce7a6c08edbfa8493359d3eea)